integration/benchmark/cmd/ci/main.go

```diff
@@ -75,42 +53,44 @@ func main() {
 
 	// CI relevant flags
 	grpcAddressFlag := flag.String("grpc-address", defaultGRPCAddress, "listen address for gRPC server")
-	initialTPSFlag := flag.Int("initial-tps", 10, "starting transactions per second")
-	maxTPSFlag := flag.Int("max-tps", *initialTPSFlag, "maximum transactions per second allowed")
+	initialTPSFlag := flag.Int("tps-initial", 10, "starting transactions per second")
+	maxTPSFlag := flag.Int("tps-max", *initialTPSFlag, "maximum transactions per second allowed")
+	minTPSFlag := flag.Int("tps-min", *initialTPSFlag, "minimum transactions per second allowed")
 	durationFlag := flag.Duration("duration", 10*time.Minute, "test duration")
+	gitRepoPathFlag := flag.String("git-repo-path", "..", "git repo path of the filesystem")
+	gitRepoURLFlag := flag.String("git-repo-url", "https://github.com/onflow/flow-go.git", "git repo URL")
 	bigQueryProjectFlag := flag.String("bigquery-project", "dapperlabs-data", "project name for the bigquery uploader")
 	bigQueryDatasetFlag := flag.String("bigquery-dataset", "dev_src_flow_tps_metrics", "dataset name for the bigquery uploader")
-	bigQueryTableFlag := flag.String("bigquery-table", "tpsslices", "table name for the bigquery uploader")
-	sliceSize := flag.Duration("slice-size", 2*time.Minute, "the amount of time that each slice covers")
+	bigQueryRawTableFlag := flag.String("bigquery-raw-table", "rawResults", "table name for the bigquery raw results")
 	flag.Parse()
 
-	// Version and Commit Info
-	gitSha := build.Commit()
-	goVersion := runtime.Version()
-	osVersion := runtime.GOOS + runtime.GOARCH
-
-	runStartTime := time.Now()
-	if gitSha == "undefined" {
-		gitSha = runStartTime.String()
-	}
-
 	chainID := flowsdk.Emulator
 
 	// parse log level and apply to logger
 	log := zerolog.New(os.Stderr).With().Timestamp().Logger().Output(zerolog.ConsoleWriter{Out: os.Stderr})
 	lvl, err := zerolog.ParseLevel(strings.ToLower(*logLvl))
 	if err != nil {
-		log.Fatal().Err(err).Msg("invalid log level")
+		log.Fatal().Err(err).Str("strLevel", *logLvl).Msg("invalid log level")
 	}
 	log = log.Level(lvl)
 
-	server := metrics.NewServer(log, metricport)
-	<-server.Ready()
-	loaderMetrics := metrics.NewLoaderCollector()
+	git, err := NewGit(log, *gitRepoPathFlag, *gitRepoURLFlag)
```

Comment:
Should repoInfo be optional?  maybe just use a default repoInfo instead of error-ing out